src/runtime/subscription-linking-flow.ts

```diff
@@ -16,21 +16,27 @@
 
 import {ActivityIframeView} from '../ui/activity-iframe-view';
 import {ActivityPorts} from '../components/activities';
+import {
+  AnalyticsEvent
+} from '../proto/api_messages';
 import {Deps} from './deps';
 import {DialogManager} from '../components/dialog-manager';
 import {
   LinkSubscriptionRequest,
   LinkSubscriptionResult,
 } from '../api/subscriptions';
 import {PageConfig} from '../model/page-config';
-import {SubscriptionLinkingCompleteResponse} from '../proto/api_messages';
+import {
+  SubscriptionLinkingCompleteResponse
+} from '../proto/api_messages';
 import {feArgs, feUrl} from './services';
 
 export class SubscriptionLinkingFlow {
   private readonly activityPorts_: ActivityPorts;
   private readonly win_: Window;
   private readonly pageConfig_: PageConfig;
   private readonly dialogManager_: DialogManager;
+  private readonly deps_: Deps;
```

Comment:
Instead of the new lines 39, 51, and 52, we can just update line 43 to make `deps_` [property parameter](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties)

Reply:
```
constructor(private readonly deps_: Deps) {
```

Reply:
Thanks for the pointer @ChrisAntaki, I'll try it out Monday.

Reply:
Updated.